tests/foreman/ui/test_ansible.py

```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
```

Comment:
Could you remove related stubbed test cases as you're automating this test here ?https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_ansible.py#L290-L323

Reply:
I have removed these two stubbed tests as they are covered in this test case.